rdt-app/src/main/assets/rule/conditional_save_calculation_rules.yml

```diff
@@ -0,0 +1,17 @@
+---
+name: step1_conditional_save
+description: Save and exit or save and continue to capture RDT form.
+priority: 1
+condition: "step1_had_fever == 'Yes' || step1_experienced_abnormal_headaches == 'Yes' || step1_felt_nausea == 'Yes'
+|| step1_has_vomited == 'Yes' || step1_experienced_diarrhea == 'Yes' || step1_experienced_chills == 'Yes'
+|| step1_felt_aches == 'Yes' || step1_had_muscle_pain == 'Yes' || step1_had_chest_pain == 'Yes' || step1_felt_abnormal_fatigue == 'Yes'
+|| step1_had_breathing_problems == 'Yes' || step1_age <= 5"
```

Comment:
Could you add the checks needed for children age `>5`?

Reply:
The checks for children over 5 are for whether to show/grey out the "poor feeding..." and "coughing..." questions which are in the relevance YAML file https://github.com/onaio/rdt-standard/blob/5b1f5994933f62f0b9d34afd65f8209894e61c81/rdt-app/src/main/assets/rule/patient_registration_relevance_rules.yml#L1-L21

Reply:
I meant that the test form should launch after some conditions are met for the <5 year old.

Those checks don't exist right?